starknet_devnet/blueprints/rpc/schema.py

```diff
@@ -169,7 +174,9 @@ def __init__(self, err: ValidationError):
         self.validation_error = err
 
     def __str__(self):
-        return f'Got invalid value for parameter: "{self.validation_error.message}"'
+        # fmt: off
+        return f"Got invalid value for parameter: \"{self.validation_error.message}\""
+        # fmt: on
```

Comment:
I do not like disabling `black` anywhere

Reply:
See my response above